packages/mux-player/src/media-theme-mux/styles.css

```diff
@@ -114,15 +114,15 @@ media-control-bar :is([role='button'], [role='switch'], button) {
   padding: 4.4% 3.2%;
 }
 
-.mxp-spacer {
+.spacer {
```

Comment:
At first, I thought this is too generic, but it should be fine because it's only inside the shadow DOM, right?

Reply:
indeed, at least some benefits thank god :P